qml/pages/LoginPage.qml

```diff
@@ -1,17 +1,17 @@
 import QtQuick 2.0
 import Sailfish.Silica 1.0
-import QtWebKit 3.0
 import harbour.sailslack 1.0 as Slack
+import Sailfish.WebView 1.0
 
-Page {
+// Using WebViewPage allows the WebView to better react to virtual keyboard.
+WebViewPage {
     id: page
 
     property string processId: Math.random().toString(36).substring(7)
     property string startUrl: "https://slack.com/oauth/authorize?scope=client&client_id=" + slackClientId + "&redirect_uri=https%3A%2F%2Flocalhost%3A3000%2Foauth%2Fcallback"
```

Comment:
Looks like there's typo in this line - the return `redirect_uri` must be `http`, not `https` - not sure how this ever worked :)

Reply:
Ooops, you're right. Looks like one of the previous commits did this.. It may be working since the redirect URL was specified by you when creating the Slack client id.